server.js

```diff
@@ -1,9 +1,20 @@
+require('dotenv').config();
 const browserSync = require('browser-sync');
 const config = require('./app/config');
 const { App } = require('./app');
 const { AuthProvider } = require('./app/authProvider');
 const { routes } = require('./app/routes');
 
+Object.keys(config).map((configKey) => {
```

Comment:
can use object.entries

Reply:
I tried that initially but doesn't recognise it here

Reply:
interesting

Reply:
probably why we use require here instead of import. babel not transpiling this file?